src/reactive_system_urc/src/ReactiveSystemNode.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * Created by William Gu on Sept 29 2018
+ * Implementation for Reactive System Node
```

Comment:
It'd be good to talk about what topic it subscribes to and what it does to them, and what topics it publishes. In other words what is the purpose of the node.